### DISCUSSION TITLE
向不存在的time series插入数据，结果正常，并未报错，这块机制是优化过了吗？

### DISCUSSION THREAD

**Discussion by ArcDevil9** [apache/iotdb#3871] [Q&A] [answered]
按照操作指导插入数据，并未有像指导书那样的报错情况，这块是已经优化了吗？
![image](https://user-images.githubusercontent.com/48848130/131310850-1e947f22-d4f2-42a8-8fb6-1aec41cfabc8.png)

向不存在的time series插入数据，结果正常，并未报错。实际上是会正常生成storage group和time series以及对应的数据
![image](https://user-images.githubusercontent.com/48848130/131311459-edcaed60-0858-481c-8da9-3ab73de3bf2a.png)

**Comment by ArcDevil9** [comment]
补充版本信息：使用的的0.11.4的released版本

**Comment by HTHou** [comment, marked as answer]
IoTDB支持根据插入的数据自动注册时间序列，因此不会报错。这里是文档错误。
另外，0.11 版本IoTDB看对应的0.11版本文档吧。
https://iotdb.apache.org/zh/UserGuide/V0.11.x/Get%20Started/QuickStart.html

**Comment by HeimingZ** [comment]
0.11.4引入了自动创建元数据功能，可查看https://iotdb.apache.org/UserGuide/Master/Data-Concept/Auto-Create-MetaData.html
